src/pivot-table/hooks/__tests__/use-head-cell-dim.test.ts

```diff
@@ -52,6 +55,20 @@ describe("UseHeadCellDim", () => {
     expect(result.current.open).toBe(false);
   });
 
+  test("handleOpenMenu should not set open state to true when isAdjustingWidth is true", async () => {
+    const result = renderer();
+
+    await act(async () => {
```

Comment:
I tried without the async here but it did not work when you have two act in a row

Reply:
Not that I think it will matter but `act` should be imported from `import { act } from "@testing-library/react";`